docs/web-links.md

```diff
@@ -0,0 +1,25 @@
+Web Links
+=========
+
+Web links are objects that point to URLs. These objects are also known as bookmarks within the Box web application.
+Web link objects are treated similarly to file objects, so they will also support shared links, copy, permanent delete,
+and restore with the format.
```

Comment:
Yes you are correct with the format shouldn't be there. I've also resolved conflicts  with the master.